test.py

```diff
@@ -61,15 +45,11 @@
     driver = Driver(options=options)
     try:
         driver.get(url)
```

Comment:
You might be able to avoid the HTML file altogether  and just use ``driver.get('about:blank')``

Reply:
Ahh, cool. Done.